courses/free-courses-ur.md

```diff
@@ -9,11 +9,14 @@
 * [HTML and CSS](#html-and-css)
 * [Java](#java)
 * [Python](#python)
+* [System Design](#system-design)
 
 
 ### Algorithms
 
+* [Complete C++ Placement DSA Course](https://www.youtube.com/watch?v=WQoB2z67hvY&list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA) - Love Babbar
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes